src/store/reducers/transactions.js

```diff
@@ -0,0 +1,40 @@
+import actionTypes from '../../constants/actions';
+
+/**
+ *
+ * @param {Array} state
+ * @param {Object} action
+ */
+const dialog = (state = { pending: [], confirmed: [] }, action) => {
+  let startTimesamp;
+
+  switch (action.type) {
+    case actionTypes.transactionAdded:
+      return Object.assign({}, state, {
+        pending: [action.data, ...state.pending],
+      });
+    case actionTypes.transactionsLoaded:
+      return Object.assign({}, state, {
+        confirmed: [
+          ...state.confirmed,
+          ...action.data,
+        ],
+      });
+    case actionTypes.transactionsUpdated:
+      startTimesamp = state && state.confirmed && state.confirmed.length ?
```

Comment:
Having in mind that we've defined the initial state of `state` equal to `{ pending: [], confirmed: [] }`. I think we just need to check `state.confirmed.length`.